src/python_pachyderm/experimental/mixin/pfs.py

```diff
@@ -4,15 +4,21 @@
 import time
 import itertools
 import tarfile
-from contextlib import contextmanager
-from typing import Iterator, Union, List, BinaryIO
 import subprocess
 from pathlib import Path
+from contextlib import contextmanager
+from typing import Iterator, Union, List, BinaryIO
 
-from python_pachyderm.pfs import commit_from, Commit, uuid_re
-from python_pachyderm.proto.v2 import pfs
-from python_pachyderm.service import pfs_proto, Service
+from python_pachyderm.experimental.pfs import commit_from, Commit, uuid_re
+from python_pachyderm.service import Service  # , pfs_proto
```

Comment:
Clarifying my [above comment](https://github.com/pachyderm/python-pachyderm/pull/360#issuecomment-988392250) and pinning this for posterity: we could, as a later optimization, import `python_pachyderm.service.pfs_proto` and then change the parts of `ModifyFileClient`'s API that take a URL, byte iterator, or anything else that isn't a protobuf so that they use the non-experimental client's API. Thus instead of:
```
        >>> with client.modify_file_client(c) as mfc:
        >>>     mfc.put_file_bytes(
        ...         "/new_file.txt",
        ...         data)
```
becoming `send(bp_to_pb(experimental.proto.PutFileRequest("/new_file.txt", data)))`, it could be `send(proto.PutFileRequest("/new_file.txt", data))` and avoid the `bp_to_pb` round trip.

Reply:
I can add this change to make the `ModifyFile` rpc route not use any betterproto objects.